### ISSUE TITLE
Failing on getting  LV UUID's

### ISSUE THREAD

**Issue by rhugga** [scality/metalk8s#412]
Just pulling my hair out trying to massage this LVM stuff, about to just rip this piece out and implement my own.

Anyone know why this is failing with ansible when the command works manually (even as a non-root user):

```
TASK [setup_lvm_lv : Setup LVM: Get UUIDs of LVM LVs] **********************************************************************************************************************************************************************************************************
Wednesday 26 September 2018  09:33:32 -0700 (0:00:00.273)       0:07:55.196 *** 
failed: [st11p01if-ztds24083901.example.com] (item={'value': {'force': False, 'lv_name': u'metalk8s_lv01', 'labels': {'scality.com/metalk8s_node': u'st11p01if-ztds24083901.example.com', 'scality.com/metalk8s_fstype': u'ext4', 'scality.com/metalk8s_vg': u'vg_metalk8s'}, 'fstype': u'ext4', 'fs_opts': u'-m 0', 'host': u'st11p01if-ztds24083901.example.com', 'mount_opts': u'defaults,noatime', 'vg_prop': {'host_path': u'/mnt/vg_metalk8s', 'pv_dict': {u'metalk8s_lv01': {u'size': u'52G'}, u'metalk8s_lv03': {u'size': u'11G'}, u'metalk8s_lv02': {u'size': u'5G'}}, 'drives': [u'/dev/sdf', u'/dev/sdg', u'/dev/sdh'], 'vg_name': u'vg_metalk8s', 'storageclass': u'local-lvm'}, u'size': u'52G'}, 'key': '/dev/vg_metalk8s/metalk8s_lv01'}) => {"changed": false, "cmd": "blkid -s UUID -o value /dev/vg_metalk8s/metalk8s_lv01", "item": {"key": "/dev/vg_metalk8s/metalk8s_lv01", "value": {"force": false, "fs_opts": "-m 0", "fstype": "ext4", "host": "st11p01if-ztds24083901.example.com", "labels": {"scality.com/metalk8s_fstype": "ext4", "scality.com/metalk8s_node": "st11p01if-ztds24083901.example.com", "scality.com/metalk8s_vg": "vg_metalk8s"}, "lv_name": "metalk8s_lv01", "mount_opts": "defaults,noatime", "size": "52G", "vg_prop": {"drives": ["/dev/sdf", "/dev/sdg", "/dev/sdh"], "host_path": "/mnt/vg_metalk8s", "pv_dict": {"metalk8s_lv01": {"size": "52G"}, "metalk8s_lv02": {"size": "5G"}, "metalk8s_lv03": {"size": "11G"}}, "storageclass": "local-lvm", "vg_name": "vg_metalk8s"}}}, "msg": "[Errno 2] No such file or directory", "rc": 2}
```

```
-127-root@st11p01if-ztds24083901:~ # blkid -s UUID -o value /dev/vg_metalk8s/metalk8s_lv01
807bf16a-5cdd-47d9-9dd9-6293c0f6847b
```

**Comment by NicolasT** [comment]
Hey hey, no need to pull any hairs or rip out anything.

The error seems to occur on host `st11p01if-ztds24083901.example.com`, while you seem to be running the command on `st13p01if-ztds07221301`, which might explain.

Please get back on #408, there may be a mistake in your vars configuration.

**Comment by rhugga** [comment]
Resolved with same resolution as #408.
